README.md

```diff
@@ -2,22 +2,36 @@
 
 ## Quick start
 
+### Use Docker
+
+Run the following command to start a new Docker container with `catadog`.
+
+```
+docker run --rm -d --name catadog ghcr.io/datadog/catadog
```

Comment:
The `--rm -d --name catadog` tags are not technically needed, but I think they're helpful, i.e. the container will automatically be removed when stopped, the user can run the container in the background, and the user can automatically directly reference the container as `catadog` instead of having to find its ID or randomized name. Should I keep the tags here or change the command to a more simple `docker run ghcr.io/datadog/catadog`?

Reply:
I think the tags are totally fine. Anyone using Docker should know what they are, and if this is the most common usage, than this is convenient to just copy-paste.